pkg/security/ptracer/cws.go

```diff
@@ -533,8 +538,7 @@ func ptrace(tracer *Tracer, probeAddr string, syscallHandlers map[int]syscallHan
 			}
 		case CallbackPostType:
 			syscallMsg, msgExists := process.Nr[nr]
-			handler, handlerFound := syscallHandlers[nr]
-			if handlerFound && msgExists && (handler.ShouldSend != nil || handler.RetFunc != nil) {
+			if msgExists {
```

Comment:
I think we must keep the `&& (handler.ShouldSend != nil || handler.RetFunc != nil)` part

Reply:
the test is already done in the following ifs, I'm not sure I see the value is keeping this additional check

Reply:
Yep you're right, I just saw it .. -_- sry ! LGTM